src/components/CardEditor/CardEditForm/CardEditFormContent.jsx

```diff
@@ -122,17 +139,7 @@ const CardEditFormContent = ({
 
   const handleTranslation = useCallback(
```

Comment:
Since the function is defined outside of the function, this useCallback and function definition is no longer needed

Reply:
I left it this way to keep it consistent with the other places that similar translation functions are used. I think perhaps we should either use `useCallback` for all of them, or none of them.

Reply:
Okay its just redundant code so we can refactor all the other spots later